chainer/link.py

```diff
@@ -161,11 +173,14 @@ def local_link_hooks(self):
 
     @property
     def _n_local_link_hooks(self):
+        # type: () -> int
+
         return (0 if self._local_link_hooks is None
                 else len(self._local_link_hooks))
 
     @property
     def xp(self):
+        # type: () -> numpy
```

Comment:
Hmm, I noticed this method also returns `cuda`. Should it be `Union[numpy, cuda]`...?

Reply:
Maybe, It returns module.